### PR TITLE
Added clear REST API audit log [#483]

### PR DESCRIPTION
**Pull request by BRUCELLA2**
## Status
**READY**

## Migrations
NO

## Description
* Added a button to clear the REST API audit log (only for Admin)

### CODE REVIEW

**Comment by BRUCELLA2** [comment]
I forgot the effects tests. I will add them.

File: comixed-frontend/src/app/backend-status/effects/clear-rest-audit-log.effects.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                  )
+                )
+              : this.alertService.error(
+                  this.translateService.instant(
+                    'rest.clear-rest-audit-log.effects.clear-audit-log.error.detail'
+                  )
+                )
+          ),
+          map((response: ApiResponse<void>) =>
+            response.success ? restAuditLogCleared() : clearRestAuditLogFailed()
```
Comment (mcpierce) [inline comixed-frontend/src/app/backend-status/effects/clear-rest-audit-log.effects.ts]: Would it be possible to have this return a mergeMap that includes the success action as well as an action to clear the loaded audit log?

**Review by mcpierce — CHANGES_REQUESTED** [review]
One question for the initial review.

**Comment by BRUCELLA2** [comment]
My pc is working properly again! So I made some modifications taking into account your remarks. I added the refresh of the rest log list after the clean. The code works but I'm not sure it's super clean because I'm not too familiar with actions, effects, reducers and selectors...

**Review by mcpierce — APPROVED** [review]
All looks good! Thanks for this!

**Pull request merged**